Examples/.bazelrc

```diff
@@ -1,2 +1 @@
 build --use_top_level_targets_for_symlinks
-build --macos_cpus=arm64,x86_64
```

Comment:
This didn't play nice with rules_xcodeproj, and since it's just an example, I felt it was fine to remove for now.